examples/kit-nextjs-article-starter/src/components/article-listing/ArticleListing.tsx

```diff
@@ -1,3 +1,5 @@
+'use client';
```

Comment:
let's try removing `use client` as much as possible.
for example, `page` can be retrieved from props: https://github.com/Sitecore/xmcloud-starter-js/pull/175